operator/validator/controller.go

```
@@ -1031,6 +1040,23 @@ func (c *controller) reportIndicesChange(ctx context.Context) bool {
 	}
 }
 
+func (c *controller) reportFeeRecipientChange(ctx context.Context) bool {
+	if c.feeRecipientChangeCh == nil {
+		c.logger.Error("fee recipient change channel not configured")
+		return false
+	}
```

Is this even possible for `feeRecipientChangeCh` to be `nil` ? I think this can only happen if we wired our code incorrectly ... so I would rather let it panic if that's the case - so we can easily discover that (rather than potentially missing this error in logs)